kafka-elasticsearch/src/main/java/com/elte/elasticSearchConsumer/elasticsearch.properties

```diff
@@ -0,0 +1,4 @@
+#i'll hide this part for now and will use it during demonstration
+hostname=anomalydetection-tes-5738607777.eu-central-1.bonsaisearch.net
```

Comment:
Is data stored in Bonsai? If yes, we are not allowed to store data in third party storage

Reply:
It's not stored in bonsai. It's only getting temporarily streamed there to further feed to Kibana for visualization.